### ISSUE TITLE
Can't download MOD44B V051 or V006

### ISSUE THREAD

**Issue by MagicForrest** [MatMatt/MODIS#21]
First of all, thanks for the great package!  I really like it and find it very useful, but right now I am having an issue.

Yesterday I downloaded the whole of MOD44B V005 with no problems, but I can't get either V051 or V006 (which are what I actually need).  Here is what happens:

```
library(MODIS)
global.extent <- extent(-180, 180, -90, 90)
getHdf(product = "MOD44B", collection = "006", extent = global.extent,  forceDownload = TRUE)
```

> No begin(-date) set, getting data from the beginning
> No end(-date) set, getting data up to the most actual
> Error in if (FtpDayDirs[1] == FALSE) { : 
>   missing value where TRUE/FALSE needed

I can see both V006 and V051with getCollection().  I am also tried the dev version of the package, no luck.  Help would be very much appreciated, TIA!

**Comment by mcdesignnl** [comment]
I have the same issue, another example:

**Comment by fdetsch** [comment]
This has been solved as of f45eabbbdc2cef5bca83fc055ea4f220dbcf4db6 and 05ba75cd2c96acb5e998c980adc7d210a2e957e0, see also details in https://github.com/MatMatt/MODIS/issues/33 (in particular the part about `?lpdaacLogin` as MOD44B is available from LP DAAC only). The list of available collections for MOD44B currently staged at LP DAAC can be retrieved via

```
getCollection("MOD44B", newest = FALSE, forceCheck = TRUE)
```

> Updating collection from LPDAAC for product: MOD44B 
> $MOD44B
> [1] "005" "051" "006"

This is currently part of the 'develop' branch only, but will be available from CRAN soon.
